README.md

```diff
@@ -84,6 +89,15 @@ This repository contains our fork of HLSDK and restored source code for Half-Lif
 
 * Clone this repostory: `$ git clone --recursive https://github.com/FWGS/xash3d-fwgs`.
 
+#### Android (Windows/Linux/macOS)
+* Install [Android Studio](https://developer.android.com/studio) (or the command line tools).
+* Install [Python](https://python.org) (at least 2.7, latest is better).
```

Comment:
I don't think it really makes sense to mention 2.7.

We support it for legacy systems but on development machine nothing stops from using latest version. I doubt that Android Studio can be ran and used on anything legacy.